lib/modules/idea-single-widgets/public/css/main.less

```diff
@@ -557,3 +557,18 @@ table.dataTable thead th, table.dataTable thead td {
 		margin-top: 0;
 	}
 }
+
+// Votes modal
+.btn.download-votes {
+	position: absolute;
```

Comment:
Zou je deze een `height` en `line-height` van 34px willen geven? Dan staat het net wat netter in de titelbalk